libs/application/templates/new-primary-school/src/fields/Review/index.tsx

```diff
@@ -169,7 +169,10 @@ export const Review: FC<ReviewScreenProps> = ({
         <CurrentSchool {...childProps} />
       )}
       <School {...childProps} />
-      <ReasonForApplication {...childProps} />
+      {applyForNeighbourhoodSchool === NO &&
+        applicationType === ApplicationType.NEW_PRIMARY_SCHOOL && (
+          <ReasonForApplication {...childProps} />
+        )}
```

Comment:
This page is always in the `NEW_PRIMARY_SCHOOL` application but only in the `ENROLLMENT_IN_PRIMARY_SCHOOL` application if `applyForNeighbourhoodSchool === NO`
```suggestion
      {{applicationType === ApplicationType.NEW_PRIMARY_SCHOOL ||
        (applicationType === ApplicationType.ENROLLMENT_IN_PRIMARY_SCHOOL &&
          applyForNeighbourhoodSchool === NO && ( (
          <ReasonForApplication {...childProps} />
        )}
```

Reply:
Fixed